sunway-college-johor-bahru.csl

```diff
@@ -149,10 +152,15 @@ TO DO
   </macro>
   <macro name="title">
     <choose>
-      <if type="chapter paper-conference article-journal article-magazine article-newspaper broadcast interview manuscript map personal_communication speech thesis map post post-weblog" match="none">
+    <!-- encyclopedia type -->
+	  <if type="entry-encyclopedia article-newspaper">
+		<text variable="title" prefix="&apos;" suffix="&apos;"/>
```

Comment:
It is better to do this with CSL quotes:

`<text variable="title" quotes="true"/>`

(see http://docs.citationstyles.org/en/stable/specification.html#quotes)